cmd/ff-proxy/main.go

```diff
@@ -389,6 +389,7 @@ func main() {
 		targetConfig  map[domain.TargetKey][]domain.Target
 		segmentConfig map[domain.SegmentKey][]domain.Segment
 		authConfig    map[domain.AuthAPIKey]string
+		approvedEnvs  map[string]struct{}
```

Comment:
Had a check in the goplayground and thinks this makes the map `nil` by default. Might be being overly paranoid here but to be safe is it possible to initialise it as well? 

I see the `len` checks on the map before we use it in the AuthRepo which should mean we're grand but just want to be extra safe in case

Reply:
Yeah should all be fine but updated to initialise it in main.go anyway